README.md

```diff
@@ -52,21 +50,19 @@ automatically removed from `production` builds:
 </article>
 ```
 
-Once you've done that you can use the `testSelector()` function to create
-a CSS/jQuery selector that looks up the right elements:
+Once you've done that you can use an attribute selectors to look up the
+elements:
 
 ```js
-import testSelector from 'ember-test-selectors';
-
 // in Acceptance Tests:
 
-find(testSelector('post-title')) // => find('[data-test-post-title]')
-find(testSelector('resource-id', '2')) // => find('[data-test-resource-id="2"]')
+find('[data-test-post-title]')
+find('[data-test-resource-id="2"]')
```

Comment:
I think this diff is actually the perfect explanation of why I think the helper was a bad idea in the first place ;)